Sources/SpeziHealthKit/HealthKit.swift

```diff
@@ -19,46 +20,46 @@ import SwiftUI
 /// The ``HealthKitConstraint/add(sample:)`` function is triggered once for every newly collected HealthKit sample, and the ``HealthKitConstraint/remove(sample:)`` function is triggered once for every deleted HealthKit sample.
 /// ```swift
 /// actor ExampleStandard: Standard, HealthKitConstraint {
-///    // Add the newly collected HKSample to your application.
-///    func add(sample: HKSample) async {
-///        ...
-///    }
-///  
-///    // Remove the deleted HKSample from your application.
-///    func remove(sample: HKDeletedObject) {
-///        ...
-///    }
+///     // Add the newly collected HKSample to your application.
+///     func add(sample: HKSample) async {
+///         // ...
+///     }
+///
+///     // Remove the deleted HKSample from your application.
+///     func remove(sample: HKDeletedObject) {
+///         // ...
+///     }
 /// }
 /// ```
 /// 
 /// Then, you can configure the ``HealthKit`` module in the configuration section of your `SpeziAppDelegate`.
 /// Provide ``HealthKitDataSourceDescription`` to define the data collection.
-/// You can, e.g., use ``CollectSample`` to collect a wide variety of `HKSampleTypes`:
+/// You can, e.g., use ``CollectSample`` to collect a wide variety of ``HealthKitSampleType``s:
 /// ```swift
 /// class ExampleAppDelegate: SpeziAppDelegate {
 ///     override var configuration: Configuration {
 ///         Configuration(standard: ExampleStandard()) {
 ///             if HKHealthStore.isHealthDataAvailable() {
```

Comment:
As far as I can see this could be removed here now that we just provide empty queries if HealthKit is not available.